### PR TITLE
Add an overlay to sleeper explaining why chemicals are uninjectable.

### PR DESCRIPTION
**Pull request by moxian**
## What Does This PR Do
Adds an overlay to the sleeper explaining why "Inject" buttons are greyed out.

## Why It's Good For The Game
On a number of occasions I shoved (and saw shoved) a badly bruised/burnt patient into a sleeper to inject saline only to see the exact saline I need being unavailable for injection. It's been very confusing every time, but given the pressure of the situation, you don't go trying to figure out why exactly this happens.

(Turns out it happens if the patient total damage is above 125. Which is dumb, honestly, and is probably a holdover from before the health rework (when we got oxydamage and stuff), but it is what it is)

## Images of changes
Graphic design is my passion. Better font (and overall) suggestions welcome.

## Changelog
:cl:
add: Sleepers get a bit of a note about being unable to inject chemicals into badly bruised people.
/:cl:

### CODE REVIEW

File: tgui/packages/tgui/interfaces/Sleeper.js [hunk 1]
```diff
@@ -236,6 +236,29 @@ const SleeperChemicals = (props, context) => {
...
+          style={{ "position": "absolute",
+            height: "220px",
+            width: "100%",
+            bottom: "0px",
+            "z-index": 120,
+            background: "rgba(153, 153, 153, 0.8)",
+          }}>
+          <Box
+            style={{ "text-align": "center",
+              transform: "rotate(20deg) translateY(100px)",
```
Comment (AffectedArc07) [inline tgui/packages/tgui/interfaces/Sleeper.js:252]:
```suggestion
              transform: "rotate(15deg) translateY(100px)",
```
Maybe a bit less

**Comment by SteelSlayer** [comment]
I like the idea, however I will say that the angled text is a little odd to me. I think might be better with plain horizontal text, but that's me. 

Also the grey box around the text stretches to the edge of the window on the right, but on the left, it doesn't quite reach all the way. Judging by your screenshot at least.

**Comment by moxian** [comment]
But also... Upon giving this some more thought, not being able to inject people is really "not good for the game", so let me fix *that* instead.
I'll leave this PR in draft state for now as a sort of backup, in the unlikely case there's a bunch of pushback there.

**Comment by Sirryan2002** [comment]
Yeah, better UI is really something to be desired here. If the buttons are graying out, there's no point in putting a gray overlay over the entire section. Maybe just have error message feedback would be more beneficial.

**Comment by moxian** [comment]
> I like the idea, however I will say that the angled text is a little odd to me. I think might be better with plain horizontal text, but that's me.

I was afraid, that it would not read well so i didn't even try that. But you're right, it's better.

> the grey box 

Noted.

Also I just noted that it doesn't interact well with dead patients, saying that they are "critical" (rather than "dead"), and confusingly not extending all the way to Eph.

All of those will be fixed if/when i reopen this back.

File: tgui/packages/tgui/interfaces/Sleeper.js [hunk 2]
```diff
@@ -236,6 +236,29 @@ const SleeperChemicals = (props, context) => {
   } = data;
   return (
     <Section title="Occupant Chemicals" flexGrow="1">
+      {
+        (!!occupant && occupant["crisis"])
+      && (
+        <Box
+          style={{ "position": "absolute",
```
Comment (Sirryan2002) [inline tgui/packages/tgui/interfaces/Sleeper.js:243]: Not a massive fan of position:absolute. This needs to be checked to see if it dynamically changes correctly when resizing the window.

**Pull request closed without merging**